packages/catalog-search/src/tests/SearchBox.test.jsx

```diff
@@ -65,48 +65,42 @@ describe('<SearchBox />', () => {
     expect(screen.queryByTestId('suggestions')).toBeNull();
   });
 
-  test('makes algolia call with correct parameters on typing in searchbox', () => {
-    jest.useFakeTimers();
+  test('makes algolia call with correct parameters on typing in searchbox', async () => {
+    const mockAlgoliaSearchHits = ('foobar', 1);
+    index.search.mockResolvedValue(mockAlgoliaSearchHits);
     renderWithSearchContext(<SearchBoxBase enterpriseSlug="test-enterprise" index={index} />);
     userEvent.type(screen.getByRole('searchbox'), TEST_QUERY);
-    jest.runAllTimers();
-    expect(index.search.mock.calls.length).toBe(1);
+    await waitFor(() => expect(index.search.mock.calls.length).toBe(1));
     expect(index.search).toHaveBeenCalledWith(
       'test query',
       {
         attributesToHighlight: ['title'],
         filters: '',
-        attributesToRetrieve: [
-          'key',
-          'content_type',
-          'title',
-          'authoring_organizations',
-          'aggregation_key',
-          '_highlightResult',
-          'program_type',
-        ],
+        attributesToRetrieve: ['*'],
```

Comment:
Is this change related to autocomplete? Do we now get all attributes instead of the limited list before?

Reply:
this change is specifically for displaying course info modals in public catalog using the hit information provided from frontend-enterprise